### DISCUSSION TITLE
[Request for help] Dealing with spam on LlamaIndex (llama_index) repo

### DISCUSSION THREAD

**Discussion by jerryjliu** [community/community#54506] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi all,

I'm the project owner for [LlamaIndex](https://github.com/jerryjliu/llama_index), a popular open-source repo for connecting LLM's and data. In the past few days we've been under attack by spambots - just take a look at the [closed issues](https://github.com/jerryjliu/llama_index/issues?q=is%3Aissue+is%3Aclosed) and you'll see that a lot of our recently closed ones are spam. There's now 50-100 spam issues every day. 

We've tried the following:
- Block individual users and report them
- Close issues
- Manually delete issues

But there's the following challenges:
- Blocking/reporting users is ineffective; new, recent user accounts pop up.
- Closing issues clutters our issues page; many of our users have noticed the spam.
- You can't delete issues in bulk - you have to manually delete each one.

Is there any help we can get from Github Support on this? E.g. force-deleting all spam issues, having stricter controls on who can submit an issue on Github, etc. Curious what the best practices are here.

**Comment by smyja** [comment]
That's a lot. Close the issues section. Django completely closed theirs and moved to something external. Mantine Ui uses the discussions section for issues.

**Comment by airtower-luna** [comment]
Yikes, that's really bad. I think the best option you have is to [set up interaction limits](https://docs.github.com/en/communities/moderating-comments-and-conversations/limiting-interactions-in-your-repository). I notice that not all of the spammers are new accounts, but by limiting to existing users you'd at least exclude *some*. Limiting to contributors is much harsher and might prevent some legitimate bug reports, but given the situation it might be needed.

I really hope someone from support see this. :confused:

**Comment by queenofcorgis** [comment]
Hi there - thank you so much for reporting this - I've escalated this to the proper teams and they're investigating now.

**Comment by jerryjliu** [comment]
Hey @queenofcorgis, thanks so much for the help. Looking forward to an update.

**Comment by jerryjliu** [comment]
hi @queenofcorgis, any updates on this? there's still a high-volume of spam in the repo (even after i modified the interaction limits)

**Reply by queenofcorgis** [reply]
Thanks for following back up - I'll let the team know.
